teaching_platform/validation.py

```diff
@@ -0,0 +1,33 @@
+from password_strength import PasswordPolicy
+from werkzeug.security import generate_password_hash
+
+from teaching_platform.users import exceptions
+from teaching_platform.users import db_handler as user_dbh
+from teaching_platform.lessons import db_handler as lesson_dbh
+from teaching_platform.projects import db_handler as project_dbh
+from teaching_platform.models import Role
+
+
+def validate_password(password):
+    policy = PasswordPolicy.from_names(length=5, uppercase=1, numbers=2, special=1, nonletters=2)
```

Comment:
I suggest moving those hardcoded values into some sort of configuration file.

Reply:
Done